app/views/webhooks/edit.njk

```diff
@@ -0,0 +1,66 @@
+{% extends "layout.njk" %}
+
+{% set heading = "Update your Webhook details" if isEditing else "Create a new Webhook" %}
+{% set submitURL = formatFutureStrategyAccountPathsFor(routes.futureAccountStrategy.webhooks.update, currentGatewayAccount.type, currentGatewayAccount.service_id, currentGatewayAccount.external_id, webhook.external_id) if isEditing else formatFutureStrategyAccountPathsFor(routes.futureAccountStrategy.webhooks.create, currentGatewayAccount.type, currentGatewayAccount.service_id, currentGatewayAccount.external_id) %}
+
+{% block pageTitle %}
+  {{ heading }} - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV.UK Pay
```

Comment:
Can we get `{{currentGatewayAccount.full_type}}` from the service rather than the gateway account?